enhancements/ingress/gateway-api-with-cluster-ingress-operator.md

```diff
@@ -622,58 +622,178 @@ will be reconciled in any namespaces, removing the need for ServiceMeshMemberRol
 namespace to manually link to, or they need to use the default ingress gateway (if enabled).
-
-If the ServiceMeshControlPlane's `spec.gateways.ingress.enabled` field is set to `true`,
-Istio creates an `istio-ingressgateway` service, in the same namespace as the control plane,
-that is a ready-to-use proxy which gateways can be manually linked to. Istio [discourages](https://istio.io/latest/docs/setup/additional-setup/gateway/)
-the use of this ingress gateway as it couples the gateway to the control plane.
-
-_To summarize, there are four ways a user could link Gateways to Ingress Gateways:_
-1. Use automated deployments
-2. Use a manual deployment, manually create a new ingress gateway service and deployment, and link to this service
-3. Use a manual deployment and link to the existing `istio-ingressgateway` ingress gateway
-4. Use a manual deployment and link to a previously created automated deployment ingress gateway
+Users can also link a Gateway to a previously created automated deployment belonging to another
+Gateway.
 
 OpenShift will not inhibit or alter the functionality of automated deployments, except
-restricting creation of Gateways to specific users (_see the automated deployment section
-in Risks and Mitigations_).
+restricting creation of Gateways to specific users (see [RBAC](#rbac)).
 
 Nor will OpenShift inhibit or alter the functionality of manual deployments. Users
 are responsible for understanding and creating links to manual deployments when creating
 Gateways.
 
-The choice between these two features depends on the user's desired sharding scheme. Manual
+The choice between automated and manual deployments depends on whether a user prefers control
+over creating their own ingress gateway deployments and services, and whether they want
+multiple Gateway objects to share a single ingress gateway deployment and service. Manual
 linking, being more expressive, can establish a many-to-one Gateway-to-Gateway-Deployment
 relationship, while automated deployments strictly establish a one-to-one
 Gateway-to-Gateway-Deployment relationship. Arguably, automated deployments are more portable
 among Gateway API implementations due to the fact manually deployments require linking an
 Istio-specific service address.
 
+#### Security Policy
+
+##### Allowing All Namespaces for Gateways
+
+Unlike OpenShift ingress, neither Gateway API nor Istio restrict Gateway objects to specific namespaces. This design
+allows for different operational groups to create and manage their own [dedicated Gateway](#dedicated-gateway-topology)
+in an application namespace rather than relying on a [shared Gateway](#shared-gateway-topology) residing in another
+namespace. This departs from the current OpenShift ingress approach of all router pods being confined to the
+`openshift-ingress` namespace. Our design will allow Gateways to be created in any namespace, provided the user has
+sufficient [RBAC](#rbac) permissions.
```

Comment:
Slightly worried this language might cause a bit of confusion with "allowing gateways across namespaces".

```suggestion
##### Gateways in Namespaces

Unlike OpenShift ingress, neither Gateway API nor Istio restrict Gateway objects to specific namespaces. This design
allows for different operational groups to create and manage their own [dedicated Gateway](#dedicated-gateway-topology)
in an application namespace rather than relying on a [shared Gateway](#shared-gateway-topology) residing in another
namespace. This departs from the current OpenShift ingress approach of all router pods being confined to the
`openshift-ingress` namespace. Our design will allow Gateways to be created in any namespace, provided the user has
sufficient [RBAC](#rbac) permissions.
```